### DISCUSSION TITLE
Minimum detect resolution

### DISCUSSION THREAD

**Discussion by jshank** [blakeblackshear/frigate#867] [Ask A Question]
I have two options for my substream, 640x360 and 352x240. At current, I'm not getting anything detected. What is the minimum image resolution for accurate person detection?

**Comment by HIT2022** [comment]
AFAIK Frigate uses a motion-based 300x300 section of the larger frame for image processing, and so the idea is to choose a resolution for which the objects you want to detect will fill up as much of that 300x300 box as possible. In your case, the target object is probably such a small part of that 300x300 section that the model is unable to return a high confidence match. So, for some use cases 640x360 will work, but you'll probably be better off with the main stream's resolution.

**Reply by mitchross** [reply]
@jshank  More context https://github.com/blakeblackshear/frigate/issues/802

**Reply by matteopizzoli** [reply]
Hi, 
In some analogic camera passing trought a dahua DVR i have the substram at 288*356 ( i remeber), nothing detected!!!
a had to switch to first stream 720x1280 maybe too much but work perfectly.
in my ip camera i use the 640*480 and works pretty well
